### PR TITLE
report region write load to PD

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Report region write load to pd, this is useful to indicate hot region.
@BusyJay @AndreMouche PTAL

### CODE REVIEW

File: src/pd/protocol.rs [hunk 1]
```diff
@@ -124,13 +124,15 @@ impl super::PdClient for RpcClient {
                         region: metapb::Region,
                         leader: metapb::Peer,
                         down_peers: Vec<pdpb::PeerStats>,
-                        pending_peers: Vec<metapb::Peer>)
+                        pending_peers: Vec<metapb::Peer>,
+                        written_bytes: u64)
```
Comment (siddontang) [inline src/pd/protocol.rs]: bytes_written

Comment (siddontang) [inline src/pd/protocol.rs]: do we need to add keys_written here?

Comment (siddontang) [inline src/pd/protocol.rs]: I think the function has many args now, maybe we will support keys_read later, so maybe we can use a Options structure.

File: src/raftstore/store/peer.rs [hunk 2]
```diff
@@ -227,6 +228,7 @@ pub struct Peer {
 
     pub written_bytes: u64,
     pub written_keys: u64,
+    pub latestn_written_bytes: LatestN,
```
Comment (siddontang) [inline src/raftstore/store/peer.rs]: why not using a u64 for the and clear it after reporting to PD?

Comment (zhangjinpeng1987) [inline src/raftstore/store/peer.rs]: This will make the statistics volatile.

Comment (siddontang) [inline src/raftstore/store/peer.rs]: the bytes will be only reset when reported to PD, I think it is ok and more simple.

Comment (zhangjinpeng1987) [inline src/raftstore/store/peer.rs]: I don't think so, in that way may can not catch the real hot region. Use the sum of latest N works like monitor CPU load.

**Pull request closed without merging**